src/app/helptext/sharing/smb/smb.ts

```diff
@@ -44,6 +44,12 @@ export const helptext_sharing_smb = {
   placeholder_timemachine: T('Time Machine'),
   tooltip_timemachine: T('Enable Time Machine backups on this share.'),
 
+  placeholder_afp: T('AFP'),
```

Comment:
@anodos325 @Mrt134 what do you think about "Legacy AFP", or "AFP (Legacy)", just to put more emphasis into it?

Reply:
Only problem that comes to mind with that is it looks a little odd in comparison with TrueNAS CORE, but that's probably not a big deal. What about "Legacy AFP Behavior", Legacy AFP Compatability", or just "AFP Compatability" ?

Reply:
I would vote for Legacy AFP Compatibility, if I will let @anodos325 pick since he is the expert, and if he agrees with a change here.

Reply:
@william-gr Any final decision on the input label text?